events/companies/views.py

```diff
@@ -150,6 +143,7 @@ def post(self, request, company_id):
         if not CompanyView.check_company_rights(request, company):
             return PERMISSION_DENIED
         new_team_data = json.loads(request.body.decode())
+        print(new_team_data)
```

Comment:
Pushing prints is not the best practice. Rather than print use logging https://docs.python.org/2/howto/logging.html